src/System.Text.RegularExpressions/ref/System.Text.RegularExpressions.cs

```diff
@@ -157,6 +157,7 @@ public partial class Regex : System.Runtime.Serialization.ISerializable
         public System.Text.RegularExpressions.RegexOptions Options { get { throw null; } }
         public bool RightToLeft { get { throw null; } }
         public static string Escape(string str) { throw null; }
+        public static bool TryEscape(ReadOnlySpan<char> str, Span<char> destination, out int charsWritten) { throw null; }
```

Comment:
I found only one use of `Escape()` in CoreFX and nothing for Unescape(). Seems the new API is useless.

Reply:
@ViktorHofer Do you have a real world example of code that is using Escape/Unescape on a performance critical path? What is the name of the customer who is going to use this API?

I do not think we should be adding Span APIs just because of we can. Majority of framework APIs are perfectly fine to allocate and return strings.

Reply:
Perhaps it is not right search request https://github.com/search?q=%22Regex.Escape%22
 - I don't see any C# sample.
The only scenario I can think of is binq/google like search: an user input and we do literal search after escaping. I can't come up with a single scenario for Unescape().

Reply:
I looked a bit on Github for use of `Unescape`. I see some that looks like invalid use eg unescaping Javascript regex out of JSON. The most valid use I can think of is where you know your regex is relatively "simple" (eg, literals and escape characters) and you want it to be displayable. That is not a very convincing use though. I really don't know what it is for.

Reply:
I looked at the usage data on API Catalog and Xamarin seems to use Regex.Escape to some extent (5-20%). I'm fine with not exposing the Span overloads if you think it isn't worth it I but would like to keep the internal changes around the MinParser which operates on Span as it's the first step towards changing the RegexParser to operate on Span instead of string which should benefit the more common APIs that require parsing.